Clinical-Genomics-Lund/gens
# Make annotation tracks pan

This PR implements the usage of unseen, or off screen, canvases for drawing the annotation tracks. The visible tracks are updated by blitting segments of the off screen canvas to the visible canvas. It prevent redundant fetching of data by caching all annotation and variant data in the browser. To reduce the rendering time the off screen canvas is only rerendered when the user approaches its end.

## Other major changes
 - drop jquery


How to test:
1.  Goto Gens dev version
2. Try zooming in on the highlighted variant with shift - drag mouse
3.  Try zooming out with ctrl - mouse click
4.  Try navigating between chromosomes by clicking them on the overview canvas
5.  Try pan tracks with the mouse
6. Try pan tracks with the buttons

Expected outcome:
The new input methods is fully working.

Review:
**Review:**
- [x] code approved by Björn Hall
